### PR TITLE
[8.0] adding general flag for disabling Monitoring for Agents (used by JobAgent)

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*Core
NEW: adding general flag for disabling Monitoring for Agents (used by JobAgent)

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Review?

File: src/DIRAC/WorkloadManagementSystem/Agent/JobAgent.py [hunk 1]
```diff
@@ -84,6 +84,8 @@ def __init__(self, agentName, loadName, baseAgentName=False, properties=None):
     def initialize(self):
         """Sets default parameters and creates CE instance"""
 
+        self.am_setOption("MonitoringEnabled", False)
```
Comment (chaen) [inline src/DIRAC/WorkloadManagementSystem/Agent/JobAgent.py:87]: Does it make sense to have a different default with respect to the `Agent` ?

Comment (fstagni) [reply src/DIRAC/WorkloadManagementSystem/Agent/JobAgent.py:87]: JobAgent(s) should not send monitoring data (maybe I have not fully understood your question...?)

Comment (chaen) [reply src/DIRAC/WorkloadManagementSystem/Agent/JobAgent.py:87]: Ah okay, I did not know always wanted to disable it. Scratch that then :-)

**Review by chaen — APPROVED** [review]

**Pull request merged**